src/persist-client/src/internal/trace.rs

```diff
@@ -1098,6 +1154,75 @@ impl<T: Timestamp + Lattice> Spine<T> {
             }
         }
     }
+
+    /// Checks invariants:
+    /// - The lowers and uppers of all batches "line up".
+    /// - The lower of the "minimum" batch is `antichain[T::minimum]`.
+    /// - The upper of the "maximum" batch is `== self.upper`.
+    /// - The since of each batch is `less_than self.since`.
```

Comment:
s/less_than/less_equal/ I think!

Reply:
ah, good catch! at least I had the actual check correct